admin/class-admin-init.php

```diff
@@ -49,6 +48,9 @@ public function __construct() {
 		add_action( 'admin_notices', [ $this, 'permalink_settings_notice' ] );
 		add_action( 'admin_enqueue_scripts', [ $this->asset_manager, 'register_wp_assets' ], PHP_INT_MAX );
 
+		$page_comments = new WPSEO_Health_Check_Page_Comments();
+		$page_comments->register_test();
```

Comment:
Shouldn't we maybe have a manager class?

```
$health_checks = new WPSEO_Health_Check_Manager();
$health_checks->register( [
    new WPSEO_Health_Check_Page_Comments(),
] );
```

Reply:
I think this is fine for now. It adds extra complexity and there is nothing to manage, we just want to register the tests. There are 4/5 other planned tests and array can do the trick later on.